src/components/NavBar/NavBar.js

```diff
@@ -1,11 +1,14 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
-import { Navbar, NavItem, Nav, OverlayTrigger, Tooltip } from 'react-bootstrap';
+import { Nav, Navbar, NavDropdown, NavItem, OverlayTrigger, Tooltip } from 'react-bootstrap';
```

Comment:
@thevaleriemack @MartyTheeMartian @Moe-Shoman How about we alphabetically order imports, as Val did here?  Makes sense to me.